mungegithub/mungers/blunderbuss.go

```diff
@@ -101,12 +104,20 @@ func (b *BlunderbussMunger) potentialOwners(issue *githubapi.Issue, files []*git
 		// makes three buckets, we shouldn't have many 10k+
 		// line changes.
 		fileWeight = int64(math.Log10(float64(fileWeight))) + 1
-		fileOwners := sets.String{}
 		if leafOnly {
-			fileOwners = b.features.Repos.LeafAssignees(*file.Filename)
+			if b.UseReviewers {
```

Comment:
@apelisse any comments here?  Changes because of rebase on top of upstream/master